music21/stream/base.py

```diff
@@ -86,7 +86,7 @@
 
 BestQuantizationMatch = namedtuple(
     'BestQuantizationMatch',
-    ['error', 'tick', 'match', 'signedError', 'divisor']
+    ['remainingGap', 'error', 'tick', 'match', 'signedError', 'divisor']
```

Comment:
okay, but generally add new attributes to a namedtuple at the end so anyone using `[0]` still gets the same result.

Reply:
True, but then we could no longer use it for sorting (unless there's something clever I'm not thinking of).

Reply:
Generally I've used namedtuples because before it was a standard (anonymous) tuple, so people might still be using access by index or destructuring paradigms.

Reply:
ah, yes that's fine.  You could sort with `key=lambda bqm: (bqm[-1], *bqm)` but yeah, too much trouble for backwards compatibility on an internal object.

I did though change it to use min -- there's no need to sort the whole thing if you're just getting the minimum

Reply:
ah! That was old code back when the m21 team didn't know its own algorithmic complexity